style.css

```diff
@@ -0,0 +1,116 @@
+.container {
+    display: flex;
+    flex-direction: column;
+}
+
+.searching {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    height: 60px;
+    background-color: #5362CE;
+}
+
+.logo-circle {
+    margin-left: 15px;
+    width: 45px;
+    height: 45px;
+    border-radius: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.logo-text {
+    font-weight: bolder;
+    color: white;
+
+}
+
+.search-bar {
+    display: flex;
+    justify-content: left;
+    padding-left: 20px; 
+    background-color: white;
+    width: 380px;
+    height: 40px;
+    border: 0px;
+    border-radius: 20px;
+}
+
+.search-bar::placeholder {
+    color: #d1d1d1;
+}
+.icon {
+    margin-left: 20px;
+}
+
+.login-signup {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    margin-right: 20px;
+    width: 160px;
+    height: 40px;
+}
+
+.login {
+    color: white;
+    width: 80px;
+    height: 30px;
+    background-color:#5362CE;
+    border: 0px;
+    border-radius: 10px;
+}
+
+.signup {
+    color: #5362CE;
+    width: 80px;
+    height: 30px;
+    background-color: white;
+    border-radius: 10px;
+    border: 0px;
+    cursor: pointer;
+}
+
+.information {
+    display: flex;  
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 300px;
+    background-color: #D9D9D9;
+}
+
+h1 {
+    margin-left: 20px;
+    font-weight: bolder;
+}
+
+.folder {
+    display: flex;
+    justify-content: space-around;
```

Comment:
Please specify class **space-around** for these styles and reuse it anywhere that uses it.